src/Cake.XdtTransform/Cake.XdtTransform.csproj

```
@@ -30,7 +30,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Cake.Core" Version="1.0.0" PrivateAssets="All" />
+    <PackageReference Include="Cake.Core" Version="2.0.0" PrivateAssets="All" />
     <PackageReference Include="DotNet.Xdt" Version="2.2.1" />
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
```

Bump other dependencies too:
```suggestion
    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
```

---

See my comment in https://github.com/phillipsj/Cake.XdtTransform/pull/70#discussion_r767144437

---

done.